core/dcschema_enum_classes.mcf

```diff
@@ -418,3 +418,16 @@ Node: dcid:ClimaticSeasonEnum
 typeOf: schema:Class
 subClassOf: schema:Enumeration
 name: "ClimaticSeasonEnum"
+
+Node: dcid:CurrencyUnitStandardizationEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Identifies whether the currency of a region is represented in the currency of the observation period (LocalCurrency), or the current currency of the region (Standardized)."
+name: "CurrencyUnitStandardizationEnum"
+
+Node: dcid:Currency
```

Comment:
nit: this can be in dcschema.mcf (not technically an enum)

Reply:
what makes something an enum in dc? I was going on the working knowledge that it's a class with a list of defined instances. This one has enum instances of "LocalCurrency" and "StandardizedCurrency"

Reply:
I'm referring to Currency class.  An enum class is a subclass of `Enumeration`.

Reply:
I see. And that Currency class can still have enum instances? I was going to add the world currencies to the dc enum instances list, which then would be members of an enum class, right? Also happy to leave Currency as a direct class if we don't want to do that anymore.

Reply:
In this case, we could start a new file (currency.mcf) which includes this class, and all the currencies that are about to be added.

Reply:
Will make Currency and its instances into its own MCF file. Is there guidance on when to start a new file vs add to existing?